ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala

```diff
@@ -12,21 +12,45 @@ class IndexedUpdatesMetrics(prefix: MetricName, metricFactory: Factory) {
   @MetricDoc.Tag(
     summary = "Number of events that will be metered",
     description = """Represents the number of events that will be included in the metering report.
-        |This is an estimate of the total number and not a substitute for the metering report.""".stripMargin,
+        |This is an estimate of the total number and not a substitute for the metering report.""",
     qualification = Debug,
   )
   val meteredEventsMeter: MetricHandle.Meter = metricFactory.meter(
     prefix :+ "metered_events",
     "Number of events that will be metered.",
   )
 
+  @MetricDoc.Tag(
+    summary = "Updated processed by the indexer",
+    description = "Represents the total number of updates processed and indexed by the indexer.",
```

Comment:
Not sure if that's accurate if you're using it in `UpdateToDbDto` because by that time the updates are not yet indexed, i.e. not yet written to the index db.

Reply:
Good point, I will change the text to reflect the fact that the updates will be indexed in the future.